scripts/uitest.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+# Define paths to uitest directories and command.
+script_dir=`dirname $0`
+root_dir=`realpath $script_dir/../`
+uitest_dir=$root_dir/uitest
```

Comment:
I think this is out of scope for the feature in the GUI. If you already have uitests elsewhere, why not using that directly? I'd like to keep things simple enough here so that when someone reports that "make uitest" failed, we are sure nothing too much exotic is going on (hence the "make dist", the pulling of latest uitest master etc).
I am not sure the result of dirname can be considered a constant.